packages/foreman/rubygem-racc/rubygem-racc.spec

```diff
@@ -0,0 +1,124 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name racc
+%global gem_require_name %{gem_name}
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.5.2
+Release: 1%{?dist}
+Summary: Racc is a LALR(1) parser generator
+Group: Development/Languages
+License: Ruby and BSD-2-Clause
+URL: http://i.loveruby.net/en/projects/racc/
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby-devel
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: tree
```

Comment:
How does it generate this dependency? Feels odd.

Reply:
Oops, that was from my debugging.